packages/change-codemod/transforms/__testfixtures__/tap-and-then-identifier-to-block.output.js

```diff
@@ -0,0 +1,34 @@
+// Tap at end of chain
+          SubscriptionProgram.where({ id: 1 })
+            .fetch({ require: false })
+            .tap((program) => {
+              expect(program.get('name')).to.equal(reqBody.name);
+            });
+
+// Tap within chain
+        SubscriptionProgram.where({ id: 1 })
+            .fetch({ require: false })
+            .tap((program) => {
+              expect(program.get('name')).to.equal(reqBody.name);
+            }).then((response) => { return console(response); });
+
+// Tap with inline expression
+          SubscriptionProgram.where({ id: 1 })
+            .fetch({ require: false })
+            .tap((program) => expect(program.get('name')).to.equal(reqBody.name));
+
+
+      request(app)
+        .post(endpoint)
+        .set('Content-Type', 'text/plain')
+        .set('Auth-Type', 'app')
+        .set('App-Id', 'fe')
+        .send(JSON.stringify(notification))
+        .expect(415)
+        .expect('Error-Type', 'UnsupportedContentType')
+    .then(result => {
+        return validateResponse(result);
+      })
```

Comment:
nit: spacing in this file

Reply:
The spacing won't be pretty after applying the codemod, so formatting with a formatter is required.  But I applied better formatting to the input so that's clearer.